flycheck-mix.el

```diff
@@ -87,13 +90,7 @@
           (shell-quote-argument (flycheck-mix-project-root))
           ))
 
-;;;###autoload
-(defun flycheck-mix-setup ()
-  "Setup Flycheck for Elixir."
-  ;; it doesn't make sense to check buffer on idle change
-  ;; only saved files will be checked by mix
-  (setq flycheck-check-syntax-automatically '(mode-enabled save))
-  (add-to-list 'flycheck-checkers 'elixir-mix))
+(add-to-list 'flycheck-checkers 'elixir-mix)
```

Comment:
Please don't remove `flycheck-mix-setup`. Apart from breaking existing configurations, it's also bad style—loading Emacs Lisp libraries should **never** have side effects, i.e. change options, etc…